### ISSUE TITLE
Remove stray console.log() in basic_analysis.test.js

### ISSUE THREAD

**Issue by humphd** [Seneca-CDOT/telescope#244]
We have some console spam in https://github.com/Seneca-CDOT/telescope/blob/master/test/basic_analysis.test.js#L24 we need to get rid of:

```
PASS test/basic_analysis.test.js

  ● Console

    console.log test/basic_analysis.test.js:24

      [Function: analyzeText]
```

**Comment by wajeehsheikh** [comment]
I can take care of this issue.

**Comment by humphd** [comment]
Fixed in #311.
